src/main/java/com/bina/varsim/util/VCFparser.java

```diff
@@ -347,7 +347,14 @@ else if (index == 9) { // Output format
                 }
             }
         }
-        Alt[] alts = string2Alt(ALT);
+        Alt[] alts = null;
+        try {
+            alts = string2Alt(ALT);
+        } catch (IllegalArgumentException e) {
+            log.warn(e.getMessage());
+            log.warn(line);
```

Comment:
Would be better to make it more informative, so append some message here.

Reply:
done. Thanks!